elixir/apps/web/test/web/live/resources/show_test.exs

```diff
@@ -97,22 +97,6 @@ defmodule Web.Live.Resources.ShowTest do
               {:live_redirect, %{to: ~p"/#{account}/resources/#{resource}/edit", kind: :push}}}
   end
 
-  test "hides edit resource button when feature is disabled", %{
```

Comment:
@AndrewDryga It seems safe to show the edit button here so that filters and name can be updated -- unless I'm missing something?

Reply:
There was nothing to edit, but yeah - we can allow editing name/filter rules now.

Reply:
👍